.codeclimate.yml

```diff
@@ -5,6 +5,9 @@ checks:
       threshold: 7
   return-statements:
     enabled: false
+  method-lines:
+    config:
+      threshold: 30
```

Comment:
What was the previous threshold - has this been increased or decreased?

Reply:
The default threshold was 25 so it has been increased. Number of lines is a bit of a crude gauge of maintainability in my opinion. However it can be a useful indicator of when a function might be getting to big, which is why I didn't switch it off.

The function that caused me to increase this threshold was this one: https://github.com/bugsnag/bugsnag-js/blob/bengourley/sessions-default-on/browser/plugins/sessions.js

Most of the lines (16 of them) are taken up by a declarative object literal. And so rather than battle with this rule, I bumped the threshold to what I consider a more reasonable level.

Reply:
30 seems reasonable to me, particularly for that example. 👍